desktop/cmp/grid/columns/Actions.js

```diff
@@ -99,7 +100,11 @@ export const actionCol = {
  *      Default small minimal buttons with an icon will be 24px
  * @returns {number} - the width in pixels
  */
-export function calcActionColWidth(count, cellPadding = Number(actionColPad), buttonWidth = 24) {
+export function calcActionColWidth(
+    count,
+    cellPadding = 5,  // This value needs to be kept in sync with $action-col-pad-px
```

Comment:
Does the variable mentioned in this comment exist?  I couldn't find it - concerned we risk driving future versions of ourselves a bit crazy... 😃

Reply:
good catch!  It was there!